README.md

```diff
@@ -1 +1,75 @@
 # CSE442-Activity2
+
+I'm jealous, I'm overzealous
+When I'm down, I get real down
+When I'm high, I don't come down
+I get angry, baby, believe me
+I could love you just like that
+And I could leave you just this fast
+
+But you don't judge me
+'Cause if you did, baby, I would judge you too
+No, you don't judge me
+'Cause if you did, baby, I would judge you too
+
+'Cause I got issues
+But you got 'em too
```

Comment:
i like this

Reply:
thanks m8 🦉

Reply:
@Proplex omg you wrote minecraft plugins too 💯

Reply:
Perfect